web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -154,6 +154,17 @@
           <span data-translate="">enableWorkflow</span>
         </a>
       </li>
+      <!--metadata workflow assistance external link(s)-->
+      <li role="menuitem"
+          data-ng-if="isMdWorkflowEnable && md.isWorkflowEnabled()
+          && isMdWorkflowAssistEnable && md.isOwned() && user.isUserAdminOrMore()" data-ng-repeat="app in workflowAssistApps">
+        <a href=""
+           data-ng-href="{{app.url.replace('{parameters}', md.getUuid())}}"
```

Comment:
Why call it {parameters} when it is a uuid?  What would happen if we decided to add {metadataId} or {language}?  In that case parameters seems like a odd placeholder.
I would recommend changing this to {uuid} since that is what is being supplied.
I would also suggest that the {language} placeholder be supplied as well as I would expect that the target application may need to know the language.  Not sure if this should be a 2 char code or a 3char language code?  Probably both would be good.